homing_missiles.py

```diff
@@ -58,20 +82,17 @@ def draw_entities(self, screen):
                                           (main_character.position_[1]-main_character.size_/2) * self.DRAWING_SCALE_,
                                           main_character.size_ * self.DRAWING_SCALE_,
                                           main_character.size_ * self.DRAWING_SCALE_)
-        pygame.draw.rect(
-            screen, self.main_character_color_, main_character_rect)
+        pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
 
     def draw_target_tags(self, screen):
         for enemy in self.universe_.enemies_.values():
             target_tag_word = self.target_tags_[enemy.ID_]
-            target_tag_label = self.ui_font_.render(target_tag_word, 1, self.enemy_color_)
+            target_tag_label = self.ui_font_.render(
+                target_tag_word, self.text_antialias_, self.enemy_color_)
             width = self.ui_font_.size(target_tag_word)[0]
-            screen.blit(target_tag_label, 
+            screen.blit(target_tag_label,
                         (enemy.position_[0] * self.DRAWING_SCALE_ - (width/2),
-                        (enemy.position_[1] * self.DRAWING_SCALE_ + enemy.size_/2) + (5 * self.DRAWING_SCALE_)))
-
-    def new_word(self):
-        return self.word_generator_.request_word(12,1)
+                        (enemy.position_[1] * self.DRAWING_SCALE_ + enemy.size_/2) + (self.target_tag_y_spacing_ * self.DRAWING_SCALE_)))
```

Comment:
Didn't know how to write it without confusing a reader. 
`(enemy.position_[0] * self.DRAWING_SCALE_ - (width/2),`
 `(enemy.position_[1] * self.DRAWING_SCALE_ + enemy.size_/2) + \`
`(self.target_tag_y_spacing_ * self.DRAWING_SCALE_)))`

I was thinking this would confuse the reader into there being 3 parameters.